gnovm/stdlibs/stdlibs.go

```diff
@@ -13,11 +13,31 @@ func GetContext(m *gno.Machine) ExecContext {
 	return libsstd.GetContext(m)
 }
 
-func NativeStore(pkgPath string, name gno.Name) func(*gno.Machine) {
+func findNative(pkgPath string, name gno.Name) nativeFunc {
```

Comment:
Shall we return a boolean indicating if something was found?:

```suggestion
func findNative(pkgPath string, name gno.Name) (nativeFunc, bool) {
```

Reply:
I made a different system here altogether, which returns a restricted NativeFunc type instead. lmk if it looks good to you.